subworkflows/local/bam_variant_calling_tumor_only_all/main.nf

```diff
@@ -113,12 +113,7 @@ workflow BAM_VARIANT_CALLING_TUMOR_ONLY_ALL {
     if (tools.split(',').contains('mutect2')) {
         BAM_VARIANT_CALLING_TUMOR_ONLY_MUTECT2(
             // Adjust meta.map to simplify joining channels
-            // joint_mutect2 mode needs different meta.map than regular mode
-            cram.map{ meta, cram, crai ->
-                joint_mutect2 ?
-                [ meta + [ id:meta.patient ] - meta.subMap('sample', 'status', 'data_type', 'patient') , cram, crai ] :
-                [ meta + [ id:meta.sample ] - meta.subMap('sample', 'status', 'data_type'), cram, crai ]
-            },
+            cram.map{ meta, cram, crai -> [ meta + [ id:meta.sample ] - meta.subMap('sample', 'status', 'data_type'), cram, crai ] },
```

Comment:
so here you assign the sample name as Id for either scenario and in the subworkflow you then switch to patient if joint. why did it not pick up the conditional before?

Reply:
Because we actually ran mpileup in regular mode

Reply:
you pileupsummaries?

Reply:
GETPILEUPSUMMARIES is run on each individual sample while MUTECT2 is ran jointly on all samples from the same patient.

Reply:
👍 good then we ar eon the same page :D